app/sections/about/about.component.ts

```diff
@@ -1,11 +1,11 @@
 import {Component} from 'angular2/core';
 import {RouteData, RouteParams} from 'angular2/router';
 import {ContentfulService} from 'ng2-contentful/dist/src/services/contentful.service';
-import {ContentfulConfig} from '../../app.constans';
 import {ROUTER_DIRECTIVES} from 'angular2/router';
```

Comment:
import {RouteData, RouteParams} from 'angular2/router';
 import {ROUTER_DIRECTIVES} from 'angular2/router'

Those can be merger